### DISCUSSION TITLE
Remove underline in Anchor component

### DISCUSSION THREAD

**Discussion by pradyuk** [twilio-labs/paste#872] [Q&A] [answered]
If I use the anchor component, I could not find a way to remove the underline when there is an href. Is there a way to do it?

**Comment by serifluous** [comment]
Can you share an example of what you're trying to achieve? The underline's needed in Anchor to give a non-color indication that it's an anchor. But we could try to help out with some alternate solutions.

**Reply by pradyuk** [reply]
Sure. My PR is [here](https://code.hq.twilio.com/twilio/console/compare/CON-5943-accLinkToDash?expand=1). In the anchor code, I'm trying to get rid of the default underline to comply with the design.

**Comment by serifluous** [comment, marked as answer]
The full discussion here should give you a solution: #596 

If that doesn't help, let us know!

**Reply by pradyuk** [reply]
Thanks @serifluous!
